components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -317,12 +322,14 @@ public int editBusinessRuleFromScratch(String uuid, BusinessRuleFromScratch busi
             for (String nodeURL : nodeList) {
                 boolean isDeployed;
                 try {
-                    isDeployed = updateDeployedArtifact(nodeURL, businessRuleFromScratch.getUuid(), deployableSiddhiApp);
+                    isDeployed = updateDeployedArtifact(nodeURL, businessRuleFromScratch.getUuid(),
+                            deployableSiddhiApp);
                     if (isDeployed) {
                         deployedNodesCount += 1;
                     }
                 } catch (TemplateManagerException e) {
-                    log.error("Deploying siddhi app for the business rule'" + businessRuleFromScratch.getUuid() + "' on " +
+                    log.error("Deploying siddhi app for the business rule'" + businessRuleFromScratch.getUuid() +
+                            "' on " +
```

Comment:
Please avoid unnecessary code line adding, you can combine both the code line together.

Reply:
Fixed by 08a2f3f42cb111e87ff88a976961497f28fbb688